.travis.yml

```diff
@@ -1,5 +1,17 @@
-language: python
-python:
-  - "2.7"
-# command to run tests
-script: nosetests
```

Comment:
These nosetests (a Python unittest discovery & running tool) should now run inside the docker image. 

This is not dependent on ROS though IIRC, so the benefit of docker may be small